Service/AuthenticationService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace YpassDesktop.Service
+{
+    public static class AuthenticationService
+    {
+        private static bool _isLoggedIn = false;
+
+        public static bool IsLoggedIn {
+            get { return _isLoggedIn; }
+            private set { _isLoggedIn = value; }
+        }
+
+        public static void Login(string password)
```

Comment:
Pas besoin de password ici vu qu'il n'est pas utilisé
```suggestion
        public static void Login()
```